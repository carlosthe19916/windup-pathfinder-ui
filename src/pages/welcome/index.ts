export { Welcome as default } from "./welcome";
