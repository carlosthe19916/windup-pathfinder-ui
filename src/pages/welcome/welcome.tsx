import React from "react";
import {
  EmptyState,
  Title,
  EmptyStateBody,
  EmptyStateVariant,
  EmptyStateIcon,
  Bullseye,
} from "@patternfly/react-core";
import { RocketIcon } from "@patternfly/react-icons";

export const Welcome: React.FC = () => {
  return (
    <Bullseye>
      <EmptyState variant={EmptyStateVariant.large}>
        <EmptyStateIcon icon={RocketIcon} />
        <Title headingLevel="h4" size="lg">
          Welcome to the Migration Toolkit for Applications
        </Title>
        <EmptyStateBody>
          The Migration Toolkit for Applications helps you assess and perform
          large-scale application migrations and modernizations. Start by
          creating a project for your applications.
        </EmptyStateBody>
      </EmptyState>
    </Bullseye>
  );
};
