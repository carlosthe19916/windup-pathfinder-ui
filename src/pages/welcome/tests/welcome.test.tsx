import React from "react";
import { shallow, mount } from "enzyme";
import { Welcome } from "../welcome";

describe("Welcome", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<Welcome />);
    expect(wrapper).toMatchSnapshot();
  });
});
