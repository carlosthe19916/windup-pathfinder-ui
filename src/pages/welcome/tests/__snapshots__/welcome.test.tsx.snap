// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Renders without crashing 1`] = `
<Bullseye>
  <EmptyState
    variant="large"
  >
    <EmptyStateIcon
      icon={[Function]}
    />
    <Title
      headingLevel="h4"
      size="lg"
    >
      Welcome to the Migration Toolkit for Applications
    </Title>
    <EmptyStateBody>
      The Migration Toolkit for Applications helps you assess and perform large-scale application migrations and modernizations. Start by creating a project for your applications.
    </EmptyStateBody>
  </EmptyState>
</Bullseye>
`;
